### PR TITLE
[python][client] Clean up samples and CI

### PR DESCRIPTION
**Pull request by robertschweizer**
Split out from https://github.com/OpenAPITools/openapi-generator/pull/17191.

See commit messages.

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh ./bin/configs/*.yaml
  ./bin/utils/export_docs_generators.sh
  ``` 
  (For Windows users, please run the script in [Git BASH](https://gitforwindows.org/))
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  IMPORTANT: Do **NOT** purge/delete any folders/files (e.g. tests) when regenerating the samples as manually written tests may be removed.
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (upcoming 7.1.0 minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

@multani @fa0311 @wing328

### CODE REVIEW

**Review by multani — APPROVED** [review]
Looks good to me, thanks @robertschweizer!

File: CI/circle_parallel.sh [hunk 1]
```diff
@@ -89,24 +89,6 @@ elif [ "$NODE_INDEX" = "3" ]; then
...
-  #sudo make altinstall
-  pyenv install --list 
-  pyenv install 3.7.12
-  #pyenv install 2.7.14 #python2 no longer supported
-  pyenv global 3.7.12
-
-  (cd samples/openapi3/client/petstore/python && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-pydantic-v1 && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-aiohttp && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-pydantic-v1-aiohttp && mvn integration-test)
```
Comment (wing328) [inline CI/circle_parallel.sh:108]: i might have missed it. where did these tests go/relocate?

Comment (robertschweizer) [reply CI/circle_parallel.sh:108]: It's in the commit message:

> [ci: Stop running Python integration tests in Circle CI](https://github.com/OpenAPITools/openapi-generator/pull/17509/commits/1a256e0b1368fb706904c616db411d3ad54a4563) 
>
> These Python clients are all tested with Github Actions already, which
> is faster, more cleanly separated and only runs on modifications.
>
> I'm not cleaning up the pydantic-v1 client, I expect it to be removed soon anyway.

Comment (wing328) [reply CI/circle_parallel.sh:108]: > These Python clients are all tested with Github Actions already, which
is faster, more cleanly separated and only runs on modifications.

from what I recall, I don't think that's the case as python petstore (fake) tests cover a lot more model tests while echo spec doesn't have those schema definitions in the echo_api.yaml

Comment (robertschweizer) [reply CI/circle_parallel.sh:108]: These four `samples/openapi3/client/petstore/python*` sample folders are all tested with Github Actions workflows in `.github/workflows`.

Comment (wing328) [reply CI/circle_parallel.sh:108]: ah I see. completely forgot that.

can you please resolve the merge conflicts and i'll merge it right away? thanks again

Comment (robertschweizer) [reply CI/circle_parallel.sh:108]: Done, thank you!

**Pull request merged**